user_tools/src/spark_rapids_tools/tools/qualx/qualx_main.py

```diff
@@ -229,8 +229,29 @@ def _predict(
     split_fn: Callable[[pd.DataFrame], pd.DataFrame] = None,
     qual_tool_filter: Optional[str] = 'stage',
 ) -> Tuple[pd.DataFrame, pd.DataFrame]:
-    results = pd.DataFrame()
-    summary = pd.DataFrame()
+    results = pd.DataFrame(
+        columns=[
```

Comment:
In the python tool, we use a config file to store constants like these. It can be loaded using the class `YAMLPropertiesContainer`. In future, we could think of having a similar config file for QualX. 


Example, 
https://github.com/NVIDIA/spark-rapids-tools/blob/b30ca6d884f9e355d49023dd4da3f7a2ab1cb679/user_tools/src/spark_rapids_pytools/resources/qualification-conf.yaml#L20-L27